<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>format_definition_Dialog</class>
 <widget class="QDialog" name="format_definition_Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>363</width>
    <height>226</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Format Definition</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>20</height>
      </size>
     </property>
     <property name="text">
      <string>Specify Necessary Information</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="minimumSize">
      <size>
       <width>300</width>
       <height>160</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>160</height>
      </size>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QComboBox" name="comboBox">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>40</y>
        <width>51</width>
        <height>20</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>X Y</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Inl Crl</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>10</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Well name</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QSpinBox" name="wel_name_spinBox">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>10</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
      <property name="specialValueText">
       <string/>
      </property>
      <property name="prefix">
       <string>col: </string>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="value">
       <number>1</number>
      </property>
     </widget>
     <widget class="QSpinBox" name="x_spinBox">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>40</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
      <property name="specialValueText">
       <string>col:</string>
      </property>
      <property name="prefix">
       <string>col: </string>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="value">
       <number>2</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>40</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Position</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QSpinBox" name="kb_spinBox">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>70</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
      <property name="specialValueText">
       <string/>
      </property>
      <property name="prefix">
       <string>col: </string>
      </property>
      <property name="value">
       <number>4</number>
      </property>
     </widget>
     <widget class="QSpinBox" name="wd_spinBox">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>100</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
      <property name="specialValueText">
       <string/>
      </property>
      <property name="prefix">
       <string>col: </string>
      </property>
     </widget>
     <widget class="QSpinBox" name="y_spinBox">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>40</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
      <property name="specialValueText">
       <string/>
      </property>
      <property name="prefix">
       <string>col: </string>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="value">
       <number>3</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>70</y>
        <width>91</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Kelly Bushing (KB)</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>100</y>
        <width>91</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Water Depth (WD)</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QSpinBox" name="td_spinBox">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>130</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
      <property name="specialValueText">
       <string/>
      </property>
      <property name="prefix">
       <string>col: </string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>130</y>
        <width>91</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Total Depth (TD)</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>format_definition_Dialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>181</x>
     <y>205</y>
    </hint>
    <hint type="destinationlabel">
     <x>181</x>
     <y>112</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
